dcc-download-server/src/main/java/org/icgc/dcc/download/server/service/FileSystemService.java

```diff
@@ -50,11 +50,12 @@
 
 public class FileSystemService {
 
-  private final Map<String, Table<String, DownloadDataType, DataTypeFile>> releaseDonorFileTypes;
-  private final Map<String, Multimap<String, String>> releaseProjectDonors;
-  private final Map<String, Long> releaseTimes;
+  private Map<String, Table<String, DownloadDataType, DataTypeFile>> releaseDonorFileTypes;
```

Comment:
Not thread-safe. If this service is used from multiple threads, there is no guarantee that updates to the field reference will be visible. I highly recommend JCIP :)